### ISSUE TITLE
Fork Cosmic Terminal and retheme 

### ISSUE THREAD

**Issue by Sprunglesonthehub** [AcreetionOS-Linux/AcreetionOSBuildSystem#7]
We need to get an ICED theme made for AcreetionOS for the cosmic terminal. Alacritty, the core of the Cosmic Terminal is great. We need to look at different approaches to making this happen @Oliveoilollie

**Comment by Oliveoilollie** [comment]
Alrighty, I'll look into it and we will talk about it in a vc chat

**Comment by Sprunglesonthehub** [comment]
I appreciate the quick response.  What are you thinking in regards of basing the terminal emulator? Cosmic or Alacritty? I feel that is the first question we need to answer. My vote is build on top of Cosmic, as most of the work is already done, and I trust them as our upstream more than a hectic open source project. But feel free to give your opinion, as I'm wanting to hear what you think. @Oliveoilollie

**Comment by Oliveoilollie** [comment]
i feel cosmic is gonna be better in the long run. our focus is to make a user friendly distribution. and i think cosmics terminal will fit that bill. yet its still gonna be almost as performant as Alacritty

**Comment by Sprunglesonthehub** [comment]
I agree with this entirely. Cosmic is the base for sure then. I am going to look at the color themeing tonight, as it doesn't seem *that* complicated from what I'm seeing.
